### PR TITLE
Added scikit-learn, pyreadline, and Pillow

### PR DESCRIPTION
**Pull request by neilhan**
The dockerfile has a few problems on my Mac os10.11.
Needed scikit-learn, and pillow to run 1_notmnist.ipynb
ipython auto complete isn't working, added pyreadline fixed it.

### CODE REVIEW

File: tensorflow/examples/udacity/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM gcr.io/tensorflow/tensorflow:latest
 MAINTAINER Vincent Vanhoucke <vanhoucke@google.com>
-RUN pip install scikit-learn
+RUN pip install scikit-learn scikit-learn pyreadline Pillow
```
Comment (jhseu) [inline tensorflow/examples/udacity/Dockerfile]: scikit-learn is duplicated here

**Comment by jhseu** [comment]
Jenkins, test this please

**Comment by neilhan** [comment]
yes, sorry, unnecessary scikit-learn. the pillow solves an error.
pyreadline is nice to have for iPhone autocomplete.

**Comment by jhseu** [comment]
Yeah, seems reasonable to me. Mind updating the change to remove the duplicated entry?

**Comment by neilhan** [comment]
yes, will do

**Comment by neilhan** [comment]
sending another one without duplicate scikit-learn

**Pull request closed without merging**